src/components/Map/index.js

```diff
@@ -55,10 +55,4 @@ class Map extends React.Component {
   }
 }
 
-function mapStateToProps(state) {
-  return {
-    config: state.config,
-  }
-}
-
-export default connect(mapStateToProps)(Map)
+export default Map
```

Comment:
We can remove this line and write `export default` to the beginning of line 12 of this file.